azure-devops/azext_devops/dev/team/credentials.py

```diff
@@ -31,10 +31,15 @@ def credential_set(devops_organization=None):
         connection = _get_vss_connection(devops_organization, credentials)
         location_client = connection.get_client('vsts.location.v4_1.location_client.LocationClient')
         try:
-            location_client.get_connection_data()
+            connection_data = location_client.get_connection_data()
         except Exception as ex2:
             logger.debug(ex2, exc_info=True)
             raise CLIError("Failed to authenticate using the supplied token.")
+        else:
```

Comment:
This `else` is for which `if` ?

Reply:
I still think we can remove this because `except` anyways returns control using `raise` but nothing v. important

Reply:
That's for try block. It would be executed if there are no exceptions raised in try block.

Reply:
+1 for removing else. 
indentation should be sufficient to achieve it.